agents-api/agents_api/clients/temporal.py

```diff
@@ -120,7 +120,7 @@ async def run_task_execution_workflow(
 
     return await client.start_workflow(
         TaskExecutionWorkflow.run,
-        args=[execution_input, start, previous_inputs],
+        args=[execution_input, start, previous_inputs[-1]],
```

Comment:
I think we should accept only one `current_input` var instead of this list. This can be confusing and lead to bugs down the road